### PR TITLE
Avoid going through HTTP proxy for requests to localhost in refine script

### LINKED ISSUE #2000: curl  and proxy
checks performed by `curl` (or `wget`) on the default interface `127.0.0.0.1` in `check_running` with `refine` shell script should never use a proxy server : the `--noproxy 127.0.0.0.1` parameter would avoid any bad network/proxy configuration.

```patch
--- refine	2019-03-28 14:43:15.710000051 +0100
+++ refine.curl	2019-03-28 15:03:26.410000074 +0100
@@ -154,7 +154,7 @@
     CHECK_STR="<title>OpenRefine</title>"
 
     if [ "$CURL" ] ; then
-        curl -s -S -f $URL > /dev/null 2>&1
+        curl --noproxy 127.0.0.1 -s -S -f $URL > /dev/null 2>&1
         if [ "$?" = "7" ] || [ "$?" = "22" ] ; then
             NOT_RUNNING="1"
         fi
@@ -167,7 +167,7 @@
 
     if [ -z "${NOT_RUNNING}" ] ; then
         if [ "$CURL" ] ; then
-            RUNNING=`curl -s $URL | grep "$CHECK_STR"`
+            RUNNING=`curl --noproxy 127.0.0.1 -s $URL | grep "$CHECK_STR"`
         elif [ "$WGET" ] ; then
             RUNNING=`wget -q -O - $URL | grep "$CHECK_STR"`
         fi    
```

I leave it to you to look for `wget`, thanks for OpenRefine :+1: , regards, lacsaP.

### PR DESCRIPTION
**Pull request by tejasbhosale17**
Fixes #2000

### CODE REVIEW

**Review by wetneb — CHANGES_REQUESTED** [review]
Thank you! The issue mentions that a similar adaptation should be done for downloads done via `wget`. Could you also take care of that? According to the manual it should be enough to define a `no_proxy=127.0.0.1` environment variable.

**Comment by tejasbhosale17** [comment]
Yes Sure. I made similar changes for wget please review.

File: refine [hunk 1]
```diff
@@ -96,23 +96,23 @@ check_running() {
...
     if [ "$CURL" ] ; then
-        curl -s -S -f $URL > /dev/null 2>&1
+        curl --noproxy 127.0.0.1 -s -S -f $URL > /dev/null 2>&1
         CURL_RETURN=$?
         if [ $CURL_RETURN -eq "7" ] || [ $CURL_RETURN -eq "22" ]  ; then
             NOT_RUNNING="1"
         fi
     elif [ "$WGET" ] ; then
-        wget -O - $URL > /dev/null 2>&1
+        wget --noproxy 127.0.0.1 -O - $URL > /dev/null 2>&1
```
Comment (wetneb) [inline refine:105]: No, the same syntax does not work for wget, as `--noproxy` is not a valid wget option.

File: refine [hunk 2]
```diff
@@ -94,25 +94,26 @@ check_running() {
...
     if [ "$CURL" ] ; then
-        curl -s -S -f $URL > /dev/null 2>&1
+        curl --noproxy 127.0.0.1 -s -S -f $URL > /dev/null 2>&1
         CURL_RETURN=$?
         if [ $CURL_RETURN -eq "7" ] || [ $CURL_RETURN -eq "22" ]  ; then
             NOT_RUNNING="1"
         fi
     elif [ "$WGET" ] ; then
-        wget -O - $URL > /dev/null 2>&1
+        wget --no-proxy -q -O - $URL_LOCAL > /dev/null 2>&1
```
Comment (wetneb) [inline refine:106]:
```suggestion
        no_proxy=127.0.0.1 wget -O - $URL_LOCAL > /dev/null 2>&1
```

File: refine [hunk 3]
```diff
@@ -94,25 +94,26 @@ check_running() {
...
         fi
     fi    
 
     if [ -z "${NOT_RUNNING}" ] ; then
         if [ "$CURL" ] ; then
-            RUNNING=`curl -s $URL | grep "$CHECK_STR"`
+            RUNNING=`curl --noproxy 127.0.0.1 -s $URL | grep "$CHECK_STR"`
         elif [ "$WGET" ] ; then
-            RUNNING=`wget -q -O - $URL | grep "$CHECK_STR"`
+            RUNNING=`wget --no-proxy -q -O - $URL_LOCAL | grep "$CHECK_STR"`
```
Comment (wetneb) [inline refine:116]:
```suggestion
            RUNNING=`no_proxy=127.0.0.1 wget -O - $URL_LOCAL | grep "$CHECK_STR"`
```

**Comment by tejasbhosale17** [comment]
@wetneb I made the suggested changes, please let me know if I need to follow any steps since this is my first open-source contribution.

**Review by wetneb — APPROVED** [review]
Thank you!

**Pull request merged**